specification/cognitiveservices/data-plane/AzureOpenAI/authoring/stable/2022-12-01/azureopenai.json

```diff
@@ -0,0 +1,1965 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure OpenAI API version 2022-12-01",
+    "description": "Authoring APIs for creating and managing fine tuned OpenAI models, and creating and managing deployments of OpenAI models.",
+    "version": "2022-12-01"
+  },
+  "paths": {
+    "/deployments": {
+      "get": {
+        "tags": [
+          "Deployments:"
+        ],
+        "summary": "Gets the list of deployments owned by the Azure OpenAI resource.",
+        "operationId": "Deployments_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionQueryParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeploymentList"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Getting all deployments owned by account.": {
+            "$ref": "./examples/get_deployments.json"
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Deployments:"
+        ],
+        "summary": "Creates a new deployment for the Azure OpenAI resource according to the given specification.",
+        "operationId": "Deployments_Create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionQueryParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The specification of the deployment including the model to deploy and the scale settings.",
+            "schema": {
+              "$ref": "#/definitions/Deployment"
+            }
```

Comment:
This body parameter is not marked as required, but the schema has required properties. This is likely an error.